### ISSUE TITLE
Add section on support for HITL?

### ISSUE THREAD

**Issue by hamishwillee** [mavlink/qgc-user-guide#116]
FlightGear and the like.

**Comment by DonLakeFlyer** [comment]
HITL is a nighmare of being incredbily hard to get it to work. In all honesty I'd rather have people not use it at all and use a SITL flavor. If I could remove it from QGC I would.

**Comment by hamishwillee** [comment]
Fair enough :-) I'll leave this here though for someone incredibly brave to take on.
